src/lib/serviceRoutines/getContextEntitiesByEntityId.cpp

```diff
@@ -49,15 +64,23 @@ std::string getContextEntitiesByEntityId
   ParseData*                 parseDataP
 )
 {
-  std::string                          entityId = (compV[0] == "v1")? compV[3] : compV[2];
-  std::string                          answer;
-  DiscoverContextAvailabilityResponse  response;
+  std::string  entityId = (compV[0] == "v1")? compV[3] : compV[2];
+  std::string  answer;
+
+  //
+  // Fill in parseDataP->dcar.res to pass to postDiscoverContextAvailability
+  //
+  EntityId                             eId(entityId, "", "");
+  std::vector<std::string>             attributeV;
+  Restriction                          restriction;
+
+  parseDataP->dcar.res.fill(eId, attributeV, restriction);
```

Comment:
Here is the 'beauty' in my new idea (that removes the needfor a new library):
1. fill in the standard request using its new 'fill' method.
2. call the standard operation
3. (sometimes) convert the standard response to convop response (fill method in convop-struct).